.python-version

```diff
@@ -1 +0,0 @@
-3.10.1
```

Comment:
Please dont delete this file - it tells pyenv what version of python we are using

Reply:
that's strange, I don't recall deleting it - wonder if it's something I'm doing wrong with my workflow